defos/src/defos.cpp

```diff
@@ -4,9 +4,6 @@
 
 #define DLIB_LOG_DOMAIN LIB_NAME
 #include <dmsdk/sdk.h>
-
-#if defined(DM_PLATFORM_OSX) || defined(DM_PLATFORM_WINDOWS) || defined(DM_PLATFORM_HTML5)
```

Comment:
Don't do this. It should still work on Android and iOS. Just add another `|| defined(DM_PLATFORM_LINUX)`

Reply:
Done, i forgot those platform...